web-ng/README.md

```diff
@@ -3,40 +3,43 @@
 ## Dev Environment Setup
 
 Install ng CLI using npm:
+
 ```
 npm install -g @angular/cli
 ```
 
-Install angular-devkit:
-```
-npm install --save-dev @angular-devkit/build-angular
-```
+Install recent version of Node.js using nvm:
 
-Upgrade Node.js to the newest version using nvm: (Angular requires Node.js version 10.9.0 or later.)
 ```
-nvm install node --reinstall-packages-from=node
+nvm install 13.1.0 --reinstall-packages-from=node
 ```
 
 ## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `ng serve` for a dev server. Navigate to http://localhost:4200/. The app
+will automatically reload if you change any of the source files.
```

Comment:
How did you do this format this? I had the [google/gts](https://github.com/google/gts) configured for this project, and now we all could do a `npx gts fix` to fix format for most of the code, but it did not fix README. Can you share how you have your formatter configured?

Reply:
I believe I did it in VS Code using the default settings in the Prettier plugin. It doesn't look like gts supports Markdown.